crane/template.go

```diff
@@ -0,0 +1,105 @@
+package crane
+
+import (
+	"fmt"
+	"github.com/michaelsauter/crane/print"
+	"os"
+	"path"
+	"regexp"
+	"strings"
+	"text/template"
+)
+
+type TemplateData struct {
+	Graph              DependencyGraph
+	TargetedContainers Containers
+}
+type TemplateDataForContainer struct {
+	Graph              DependencyGraph
```

Comment:
For `Dependencies`, do you mean to create an interface mapping the current struct? Called `Dependencies` as well?

And for `Graph`, I think I understand your point… maybe `DependencyMap`?